Makefile

```diff
@@ -118,9 +116,6 @@ format: $(GOIMPORTS)
 $(GOX):
 	(cd /; GO111MODULE=on go get -u github.com/mitchellh/gox)
 
-$(GOLANGCI_LINT):
-	(cd /; GO111MODULE=on go get -u github.com/golangci/golangci-lint/cmd/golangci-lint)
```

Comment:
If you want to specify the version you can do the following...

```
(cd /; GO111MODULE=on go get -u github.com/golangci/golangci-lint/cmd/golangci-lint@v1.21.0)
```

This will fetch it at a version and install it.

Reply:
That would be a less invasive approach. I like that. Let me try that instead.

Reply:
nope... resulted in the same issue. Back to the original proposal for now.